content/en/docs/configuration/externalloadbalancer.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "External Load Balancer"
+linkTitle: "External Load Balancer"
+weight: 40
+type: "docs"
+---
+
+When you are using an external load balancer provided by any host, you can face several configuration issues to get it work with Cert Manager.
+
+This documentation is meant to help in configure HTTP-01 type of challenges for instances behind external load balancer.
+
+## NAT Loopback / Hairpin
+
+The first configuration point is NAT loopback. You can face `pre check` issues due to Load Balancer preventing instances behind him to access his external interface.
+
+Some Network Load Balancer have this kind of limitation for several reasons. It can be configured through iptables rerouting configuration known as `NAT loopback`.
+
+To check if you are facing this problem :
+
+1. Check that the challenge endpoint is publically accessible : `curl <endpoint>`
+2. Check that the challenge endpoint is NOT accessible from inside behind the Load Balancer : Use SSH to open a session on a node places behind the LB ; then launch the same command than before : `curl <endpoint`
+
+The HTTP-01 challenge endpoint can be found in the logs when the `pre-check` fails. If it does not appear in the logs, you can check the challenge URL by `kubectl`command.
+
+`<endpoint>` is the URL used to test the HTTP-01 from the certificate `Issuer`. For Let's Encrypt for example, the URL is formed like <domain>/.wellknown/<hash>
+
+
+## Load Balancer HTTP endpoints
+
+If you are using a Load Balancer, you should be able to configure the Load Balancer protocol as HTTP, HTTPS, TCP, UDP. Several Load Balancer now offer free SSL certificates with Let's Encrypt.
+
+When using HTTP(s) protocols for your Load Balancer, it can intercept the challenge URL to replace the response's verification hash with their hash.
```

Comment:
do you have an example of a provider doing this? this is just plain wrong according to the ACME spec and should be reported to these services!

Reply:
@nd2014-public would be nice, if you also have something like a form link?
Do you also have config to replicate this? looking into the OVH Kubernetes service now and can only seem to keep in in TCP mode not HTTP(S) mode

Reply:
This is the conversation where _az helped me about this problem : 

https://community.letsencrypt.org/t/first-certificate-behind-an-external-load-balancer/136428

I used a LB from OVH, not a managed Kubernetes cluster from OVH.

Reply:
oh okay from what I understood these are deprecated?

Reply:
Not sure we are talking about the same thing.

I use dedicated servers from OVH and a Load Balancer. I am using Cert-manager with Rancher on hybrid cluster from multiple hosts. The Load Balancer can be configured using TCP, UDP, HTTP, HTTPS… Firstly I used HTTP + HTTPs frontends and this provoked the bug with the response interception.

Using TCP/UDP protocols solved the problem.

Reply:
Okay I was assuming you used the OVH Kunernetes service. Just looked at all producs... this LB service is quite hidden :D